environment.yml

```diff
@@ -8,7 +8,6 @@ dependencies:
   - numpy
   - astropy
   - matplotlib-base
-  - sphinx
```

Comment:
Is it intentional to remove sphinx?

Reply:
sphinx is still there, it was twice in the environment file.